cli/types/cypress.d.ts

```diff
@@ -1589,19 +1589,19 @@ declare namespace Cypress {
      *
      * @see https://on.cypress.io/nextuntil
      */
-    nextUntil<K extends keyof HTMLElementTagNameMap>(selector: K, options?: Partial<Loggable & Timeoutable>): Chainable<JQuery<HTMLElementTagNameMap[K]>>
+    nextUntil<K extends keyof HTMLElementTagNameMap>(selector: K, filter?: string, options?: Partial<Loggable & Timeoutable>): Chainable<JQuery<HTMLElementTagNameMap[K]>>
     /**
      * Get all following siblings of each DOM element in a set of matched DOM elements up to, but not including, the element provided.
      *
      * @see https://on.cypress.io/nextuntil
      */
-    nextUntil<E extends HTMLElement = HTMLElement>(options?: Partial<Loggable & Timeoutable>): Chainable<JQuery<E>>
+    nextUntil<E extends Node = HTMLElement>(selector: string, filter?: string, options?: Partial<Loggable & Timeoutable>): Chainable<JQuery<E>>
```

Comment:
can we add a couple of type tests to ensure only selector, selector + filter, and selector + options, and selector + filter _ options are applied without error?

Reply:
@emilyrohrbough Sure thing. Do we have a place we're putting those sorts of tests? I don't recall seeing them but I don't spend much time in this area

Reply:
These guys would live somewhere in here: https://github.com/cypress-io/cypress/blob/develop/cli/types/tests/cypress-tests.ts
They aren't always added but some of the more complicated one I think it's helpful to add tests for.